jobs/config.json

```diff
@@ -7648,7 +7648,7 @@
       "--provider=gke",
       "--test_args=--ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\] --kubectl-path=../../../../kubernetes_skew/cluster/kubectl.sh --minStartupPods=8",
       "--timeout=900m",
-      "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=v1.8.6-gke.0 --upgrade-image=gci"
+      "--upgrade_args=--ginkgo.focus=\\[Feature:ClusterUpgrade\\] --upgrade-target=v1.9.1-gke.0 --upgrade-image=gci"
```

Comment:
Out default version is still 1.7. So this will be an upgrade test of 1.7 to 1.9, which won't work.

Reply:
do you want to pin it to 1.8 for now?